runtime/compiler/runtime/Listener.hpp

```diff
@@ -37,12 +38,33 @@
  
     The current implementation does not provide code for nicely terminating the listener thread.
  */
+
+#define OPENJ9_LISTENER_POLL_TIMEOUT 10 // in milliseconds
```

Comment:
Should we increase this to 100 ms?

Reply:
`10 ms` was just a random value. We can set the timeout to 100 ms. The larger it is, the lesser it would wake up to check the flag.